oak_runtime/src/node/http/tests.rs

```diff
@@ -377,13 +377,12 @@ fn create_server_node(runtime: &RuntimeProxy, port: u32) -> oak_abi::Handle {
         })),
     };
 
+    // TODO(#1631): When we have a separate top for each sub-lattice, this should be changed to
+    // the top of the `user` sub-lattice.
```

Comment:
```suggestion
    // the top of the "public key identity" sub-lattice.
```

I think. See

https://github.com/project-oak/oak/blob/9fd16d81c0594264eb9bc0939b3a0a0aff80448d/oak_abi/proto/label.proto#L88-L94

Reply:
Changed to "identity sub-lattice", seeing that we might support other identities in future.